composer.json

```diff
@@ -23,6 +23,7 @@
     "prefer-stable": true,
     "require": {
         "php": "^8.0",
+        "cypresslab/gitelephant": "^4.5",
```

Comment:
This package use GNU Lesser General Public License v3.0, seems, we don't can use it.

Reply:
Why not?

Reply:
> Why not?

I dont sure, but seems we don't can distribute package with BSD-3-Clause license if use packages with LGPL license.

Reply:
Yes, better to avoid GPL if possible.